docs/docs/eslint.md

```diff
@@ -31,59 +29,155 @@ rm .prettierrc
 
-    // options to emulate prettier setup
-    semi: ["error", "never"],
-    "max-len": ["error", { code: 80 }],
-    "template-curly-spacing": ["error", "always"],
-    "arrow-parens": ["error", "as-needed"],
+    // https://www.npmjs.com/package/eslint-plugin-jsx-a11y#supported-rules
+    "jsx-a11y/accessible-emoji": "warn",
+    "jsx-a11y/anchor-is-valid": "warn",
+    "jsx-a11y/alt-text": "warn",
 
-    // standard.js
-    "space-before-function-paren": [
+    // prettier - gatsby-starter-default options
+    "prettier/prettier": [
       "error",
       {
-        named: "always",
-        anonymous: "always",
-        asyncArrow: "always",
+        printWidth: 80,
+        tabWidth: 2,
+        useTabs: false,
+        semi: false,
+        singleQuote: false,
+        endOfLine: "lf",
+        jsxSingleQuote: false,
+        trailingComma: "es5",
+        bracketSpacing: true,
+        jsxBracketSameLine: false,
+        arrowParens: "avoid",
       },
     ],
+  },
+}
+```
 
-    // standard plugin - options
-    "standard/object-curly-even-spacing": ["error", "either"],
-    "standard/array-bracket-even-spacing": ["error", "either"],
-    "standard/computed-property-even-spacing": ["error", "even"],
-    "standard/no-callback-literal": ["error", ["cb", "callback"]],
+### Understanding the Plugins
 
-    // react plugin - options
-    "react/jsx-uses-react": "error",
-    "react/jsx-uses-vars": "error",
-  },
-  parser: "babel-eslint",
-  parserOptions: {
-    ecmaVersion: 8, // optional, recommended 6+
-  },
+In the example configuration file above we explored a few plugins, as well as *extending* their recommended rule sets. Now we will discover the purpose of including these plugins and their value.
+
+#### eslint-plugin-react
+
+The React plugin assists your development experience with Gatsby by helping you remember to validate props, include default props, importing React any time JSX is used, and enforce handler naming convention (e.g. `handleClick()`). It requires a version to be specified, therefore can even prevent usage of deprecated implementations. Since Gatsby development utilizes React this is an excellent addition to accelerate your experience by bypassing errors before they're made.
+
+#### eslint-plugin-jsx-a11y
+
+> In short, A11Y means “accessibility”
+
+The [A11Y Project](https://a11yproject.com/) focuses in reinforcing accessibility in your code, ensuring your new Gatsby project can be used effectively by those with visual or auditory impairment. This plugin helps you remember to include alternative text to image tags, [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) props, and eliminating redundant role properties. Accessibility is often overlook, and by doing so can hinder the ability to provide a phenomenal user experience. Including this plugin and its recommended rule set reduces the time required to implement accessibility by reminding you throughout development.
```

Comment:
While I love the A11y Project, it isn't related to the eslint a11y plugin. A better thing to link to would be the plugin docs:
```suggestion
The [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y) tool is an accessibility linter for your code, helping you develop more inclusive Gatsby projects. This plugin encourages you to include alternative text for image tags, validates [ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) props, and eliminates redundant role properties, among other things. 

Accessibility is often overlooked, and by developing with it in mind we can provide a phenomenal user experience. Including this plugin and its recommended rule set reduces the time required to implement accessibility by reminding you throughout development.
```